app/containers/Root.js

```diff
@@ -81,18 +83,18 @@ class Root extends React.Component {
                   exact
                   path="/onboarding"
                   render={() => (
-                    <Modal withClose={hasWallets} onClose={() => history.push('/home')}>
+                    <ModalOverlay withClose={hasWallets} onClose={() => history.push('/home')}>
```

Comment:
Currently modals are re-rendered because of this `onClose`, even if nothing else has changed. Could you extract this ?

Reply:
Updated.